README.md

```diff
@@ -14,19 +14,37 @@ There are numerous bugs and half working implementations. Pull requests are grea
 
 Packages are available for Windows, Linux, and MacOS.
 
-Windows packages are available as an installer or a stand-alone zip file on the [release](https://github.com/dail8859/NotepadNext/releases) page. The installer provides additional components such as an auto-updater and Windows context menu integration. You can easily install it with Winget:
+## Windows
+
+Windows packages are available as an installer or a stand-alone zip file on the [release](https://github.com/dail8859/NotepadNext/releases) page. The installer provides additional components such as an auto-updater and Windows context menu integration. You can easily install it with `Winget`:
 
 ```powershell
 winget install dail8859.NotepadNext
 ```
 
+or `Scoop`:
```

Comment:
I have no idea what scoop is. If it's available there that's fine, but I don't maintain that or know how to address any issues if something were to be wrong. So please remove this section.

Reply:
As you wish.